### PR TITLE
Add new shapes to scatter repertoire

### PR DESCRIPTION
**Pull request by ebb-tide**
Add tritip, hollow square, ex, and plus to scatter plot shapes
correct the placement of the circle.

### CODE REVIEW

File: src/utils/drawShapes.ts [hunk 1]
```diff
@@ -20,22 +17,104 @@ export const drawSquare = (
   centerY: number,
   s?: number
```
Comment (chnn) [inline src/utils/drawShapes.ts]: What do you think about making these parameters with defaults?

```suggestion
  size: number = 4
```

Doesn't matter much at all, but might make reading the function signature slightly nicer.

File: src/utils/getSymbolScale.ts [hunk 2]
```diff
@@ -8,7 +8,14 @@ export const getSymbolScale = (table: Table): Scale<string, SymbolType> => {
   const symbolCol = table.getColumn(SYMBOL, 'string')
   const uniqueValues = Array.from(new Set(symbolCol))
 
-  const symbolTypes = ['circle', 'triangle', 'square'] as SymbolType[]
+  const symbolTypes = [
```
Comment (chnn) [inline src/utils/getSymbolScale.ts]: I think
```suggestion
  const symbolTypes: SymbolType[] = [
```
would remove the need for the type assertion.

File: src/utils/drawShapes.ts (hunk 1 above)
Comment (ebb-tide) [reply src/utils/drawShapes.ts]: that is better, thanks

**Comment by ebb-tide** [comment]
I will add those two as issues

**Pull request merged**